### DISCUSSION TITLE
How to Force <ScrollArea /> to take it's parent height ?

### DISCUSSION THREAD

**Discussion by thevipinmishra** [mantinedev/mantine#2331] [Q&A]
I'm using <ScrollArea /> at a lot of places in my dashboard, one thing I would really love to have is the ability to have automatic height (of parent) in <ScrollArea /> component, instead of adding hardcoded height like `300px`.

I've been using `useElementSize` so far to do the tricky calculations, but is there a way it can be achieved automatically ?

**Comment by thevipinmishra** [comment]
By the way, this worked & I don't know why!!!!!

My parent is a flex container, and flex-direction is column.

and `<ScrollArea>` is a direct children.

` <ScrollArea sx={{ flexGrow: 1, height: "0px" }} />`

**Reply by rtivital** [reply]
This is the correct way

**Reply by thevipinmishra** [reply]
@rtivital any idea why it doesn't works without `height: "0px" ` or any value in height?

Simple adding `flexGrow: 1` doesn't works.

**Reply by rtivital** [reply]
I do not know, usually flex: 1 works fine, example – https://codesandbox.io/s/solitary-water-b80lvp?file=/src/App.tsx
